### ISSUE TITLE
Mentorship Opportunity for CCC Projects

### ISSUE THREAD

**Issue by Salkimmich** [veraison/services#238]
### Mentorship Opportunity for CCC Projects

I'm reaching out to let you know that the Linux Foundation can fund three-month mentorships to support technical and documentation efforts for CCC projects. This is an excellent opportunity to enhance project development and attract new contributors.

#### Benefits and Flexibility
- **Flexible Timing:** Start the mentorship at a time that fits your project's schedule.
- **Enhance Your Project:** Accelerate development and increase project visibility.

#### Getting Started
1. **Draft Your Project Idea:** Utilize the [Project Idea Template](https://github.com/confidential-computing/governance/blob/main/mentoring/PROJECT_IDEA_TEMPLATE.md) to clearly outline what mentees will work on.
2. **Submit Your Idea:** Use our [Mentoring Guidelines](https://github.com/confidential-computing/governance/tree/main/mentoring) to navigate the submission process.

**Have Questions?**
Feel free to ask any mentorship-related questions right here in this issue thread for a prompt response from CCC. You can also join the CCC Slack channel #mentorship_program for general mentorship topics.

**Comment by thomas-fossati** [comment]
Thanks @Salkimmich!

We are in the process of preparing a proposal and will submit it shortly.

Is there a limit to the number of proposals that can be submitted?

**Comment by Salkimmich** [comment]
That is an excellent question. I'll discuss it with the TAC tomorrow and let you know!

**Comment by Salkimmich** [comment]
Yes! You can have more than one LF mentee at once, and that might even be a great way to save time and energy getting onboarded to this project so I think it's a good idea - but they should have independently scoped projects.

**Comment by thomas-fossati** [comment]
> Yes! You can have more than one LF mentee at once, and that might even be a great way to save time and energy getting onboarded to this project so I think it's a good idea

Thanks @Salkimmich 

> but they should have independently scoped projects.

noted.
